### PR TITLE
[15.0][ADD] base_partner_company_group

### PR DESCRIPTION
**Pull request by yankinmax**
The idea is to split `partner_company_group` to several modules depending on each of core modules. This is needed to avoid installing unwanted dependencies on the project, for example we need `partner_company_group`, but without additional installed `account`, `sale` or `crm`.
So, finally we want it to be:

1. `base_partner_company_group` -> https://github.com/OCA/partner-contact/pull/1244
2. `account_partner_company_group` depending on `base_partner_company_group` and `account` -> https://github.com/OCA/partner-contact/pull/1246
3. `crm_partner_company_group` depending on `base_partner_company_group` and `crm` -> https://github.com/OCA/partner-contact/pull/1245
4. `sale_partner_company_group` depending on `base_partner_company_group` and `sale` -> https://github.com/OCA/partner-contact/pull/1247

### CODE REVIEW

File: partner_company_group/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Camptocamp SA
```
Comment (sebalix) [inline partner_company_group/__manifest__.py]: This addon should be removed

File: base_partner_company_group/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+This Module adds a company group field to companies, which is also propagated to contacts and allows to search and group for the company group in contact, leads, sale orders and invoices.
```
Comment (sebalix) [inline base_partner_company_group/readme/DESCRIPTION.rst]: This description is wrong now. We should invite the user to check the other addons to get the feature enabled on leads, SO and invoices.

File: partner_company_group/__manifest__.py (hunk 1 above)
Comment (yankinmax) [reply partner_company_group/__manifest__.py]: **UPDATED**

File: base_partner_company_group/readme/DESCRIPTION.rst (hunk 2 above)
Comment (yankinmax) [reply base_partner_company_group/readme/DESCRIPTION.rst]: **UPDATED**

**Comment by yankinmax** [comment]
@gurneyalex if it's ok we can proceed further with splitting.
And @sebalix proposed to name modules like that:
`base_partner_company_group`, `account_partner_company_group`, `crm_partner_company_group`, `sale_partner_company_group` accordingly

**Comment by gurneyalex** [comment]
> @gurneyalex if it's ok we can proceed further with splitting. And @sebalix proposed to name modules like that: `base_partner_company_group`, `account_partner_company_group`, `crm_partner_company_group`, `sale_partner_company_group` accordingly

That's ok with me

**Review by gurneyalex — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

**Comment by simahawk** [comment]
@yankinmax pls the next time rename the module with `git move` instead of adding a new one and deleting the other :wink:

**Pull request merged**